### PR TITLE
Cancellation improvements

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb [hunk 1]
```diff
@@ -160,6 +164,16 @@ def trigger(*args)
       run_async(*args)
     end
 
+    def detect_capabilities
```
Comment (ofedoren) [inline lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb:167]: It implies multiple different capabilities, but this method only checks for `unshare` support :/

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka, LGTM, just a naming concern :/

**Review by pondrejk — APPROVED** [review]
checked on patched stream, fixes the original issue

File: lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb:167]: Eventually there might be more, but I can rename it if you feel strongly about it

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka, I have nothing to add and thanks @pondrejk for testing out.

**Pull request merged**